### PR TITLE
remame vm native hash fnc nomenclature

### PR DESCRIPTION
**Pull request by 0xKanekiKen**
## Describe your changes
This PR addresses #590. 

## Checklist before requesting a review
- Repo forked and branch created from `next` according to naming convention.
- Commit messages and codestyle follow [conventions](./CONTRIBUTING.md).
- Relevant issues are linked in the PR description.
- Tests added for new functionality.
- Documentation/comments updated according to changes.

### CODE REVIEW

File: air/src/stack/op_flags/mod.rs [hunk 1]
```diff
@@ -798,10 +798,10 @@ impl<E: FieldElement> OpFlags<E> {
 
     // ------ Degree 6 non u32 operations  --------------------------------------------------------
 
-    /// Operation Flag of RPPERM operation.
+    /// Operation Flag of HMPERM operation.
```
Comment (grjte) [inline air/src/stack/op_flags/mod.rs:801]: nit: typo in `HPerm`

**Review by grjte — CHANGES_REQUESTED** [review]
Thanks @0xKanekiKen! Looks good to me, except for one typo

**Review by itzmeanjan — APPROVED** [review]
Looks good.

**Comment by bobbinth** [comment]
@0xKanekiKen - it seems like this does not fully address #590 yet (the `hash` instruction is not added). Is the plan to add it in a different PR?

**Comment by grjte** [comment]
> @0xKanekiKen - it seems like this does not fully address #590 yet (the `hash` instruction is not added). Is the plan to add it in a different PR?

I think it makes sense to handle the renaming in this PR and add `hash` in another one, unless you object to that

**Review by grjte — APPROVED** [review]
This looks good to me for the renaming of `hperm` and `hmerge`. As mentioned by @bobbinth, the remaining item from #590 is to add the `hash` instruction. 

That task is described as requiring migration of `rescue_prime::hash_1to1` from the standard library into the assembly operations and renaming it to `hash`. However, the PR for adding that hash function to the stdlib (#576) is still pending, so it makes sense to tackle this separately (and perhaps just change the outstanding PR)

**Comment by 0xKanekiKen** [comment]
Thanks @grjte for mentioning it. I was waiting for an update on the PR #576.

**Pull request merged**